app/src/main/java/com/cooper/wheellog/WheelData.java

```diff
@@ -702,8 +727,24 @@ private void raiseAlarm(ALARM_TYPE alarmType, Context mContext) {
                 pattern = new long[]{0, 500, 100, 100, 100, 500, 100, 100, 100, 500, 100, 100, 100};
                 mCurrentAlarmExecuted = true;
                 break;
+            case GENERAL:
+                if (mp.isPlaying() == false) {
+                    mp = MediaPlayer.create(mContext, R.raw.bicycle_bell);
```

Comment:
Why bicycle_bell? I think it should be separate sound, something like inmotion sound.

Reply:
I have a //TODO to change it. Will do, use the SCV default sound for that.